imagenet/main.py

```diff
@@ -63,6 +69,21 @@ def main():
     global args, best_prec1
     args = parser.parse_args()
 
+    if args.seed is not None:
+        random.seed(args.seed)
+        torch.manual_seed(args.seed)
```

Comment:
`torch.manual_seed(args.seed)` will also seed CUDA, so you dont have to add `torch.cuda.manual_seed_all(args.seed)`